### DISCUSSION TITLE
Null driver

### DISCUSSION THREAD

**Discussion by olivervo** [spatie/laravel-newsletter#295] [Q&A]
According to `config/newsletter.php:7` "log" or "null" may be used to prevent calling the API directly from your environment. However, when I set the driver to "log" or "null" I receive error `class name must be a valid object or string.`

It seems `NullDriver.php` was removed in v5 yet the comment in `config/newsletter.php` is unchanged. Has anyone managed to disable the API using this method or was it removed in v5?

**Comment by Braunson** [comment]
I just [created](https://github.com/spatie/laravel-newsletter/issues/296#issuecomment-1576948416) NullDriver locally since it was removed.
